1-setup-windows-msys2.sh

```diff
@@ -28,25 +28,26 @@ libtool \
 make \
 patch \
 tar \
-$MINGW_PACKAGE_PREFIX-gcc \
+$MINGW_PACKAGE_PREFIX-boost \
+$MINGW_PACKAGE_PREFIX-cairomm \
```

Comment:
There is no need of explicit `cairomm` anymore :)

Reply:
Done